tests/DependencyInjection/SchedulerBundleConfigurationTest.php

```diff
@@ -84,12 +84,44 @@ public function testConfigurationCanDefineChainedTasks(): void
         self::assertArrayHasKey('random', $configuration['tasks']);
         self::assertCount(2, $configuration['tasks']['random']['tasks']);
         self::assertSame('chained', $configuration['tasks']['random']['type']);
+        self::assertSame('priority', $configuration['tasks']['random']['execution_mode']);
         self::assertSame('foo', $configuration['tasks']['random']['tasks'][0]['name']);
         self::assertSame('shell', $configuration['tasks']['random']['tasks'][0]['type']);
         self::assertSame('bar', $configuration['tasks']['random']['tasks'][1]['name']);
         self::assertSame('command', $configuration['tasks']['random']['tasks'][1]['type']);
     }
 
+    public function testConfigurationCanDefineChainedTasksSpecificExecutionMode(): void
+    {
+        $configuration = (new Processor())->processConfiguration(new SchedulerBundleConfiguration(), [
+            'scheduler_bundle' => [
+                'transport' => [
+                    'dsn' => 'cache://app',
+                ],
+                'tasks' => [
+                    'random' => [
+                        'type' => 'chained',
+                        'execution_mode' => 'first_in_first_out',
+                        'tasks' => [
+                            'foo' => [
+                                'type' => 'shell',
+                                'command' => ['ls', '-al'],
+                                'expression' => '* * * * *',
+                            ],
+                            'bar' => [
+                                'type' => 'command',
+                                'command' => 'cache:clear',
+                                'expression' => '* * * * *',
+                            ],
+                        ],
+                    ],
+                ],
+            ],
+        ]);
+
+        self::assertSame('first_in_first_out', $configuration['tasks']['random']['execution_mode']);
```

Comment:
Could be good to ensure that each "sub-task" does not have a value for `execution_mode`.

Reply:
it's true, it would even be nice to improve the configuration definition by adding checks and validations on the tasks arrayNode
https://github.com/Guikingone/SchedulerBundle/blob/0efd88bdd39c04dda6f9f821465f70f1305ea15a/src/DependencyInjection/SchedulerBundleConfiguration.php#L67